_posts/2015-08-11-tsuru-images-with-packer.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+status: publish
+published: false
+title: tsuru easier to test (spoiler: packer is awesome).
+author:
+  display_name: Tarsis Azevedo
+  url: https://github.com/tarsisazevedo
+date: '2015-08-11 14:30:00 -0300'
+date_gmt: '2015-08-11 14:30:00'
+categories:
+- Images
+tags:
+- images packer
+comments: []
+---
+
+One of our goals is make tsuru easy to people who wants to try it for
+the first time. To do it we are creating all-in-one aws and virtualbox vagrant
+box images of tsuru with [packer](https://packer.io). These images are based on
+stable and nightly packages.
+
+## packer + tsuru now
+
+Tsuru all-in-one images are builded as amazon ami and as vagrant virtualbox
+box. These images are generated with [tsuru-now script](https://github.com/tsuru/now) configured to have the python platform installed by
+default and pre-receive deploy method using archive server.
+
+Amazon amis are builded in **us-east-1 region** and you can find it in
+community amis as tsuru-stable or tsuru-nightly,
+or get the latest ami id for [stable](https://s3.amazonaws.com/tsuru-images/nightly-ami-id) and [nightly](https://s3.amazonaws.com/tsuru-images/nightly-ami-id)
```

Comment:
Amazon AMIs are available in the **us-east-1 region** and you can find
	them in the "Community AMIs" tab when launching a new instance. It's
	also possible to get the last AMIs programatically by downloading two
	files from S3:

	--- make this a block of code, indented ---
	$ curl https://s3.amazonaws.com/tsuru-images/nightly-ami-id
	ami-1350d678
	$ curl https://s3.amazonaws.com/tsuru-images/stable-ami-id
	ami-a98527c2
	--- end of indented block of code ---

Also add an extra blank line here to ensure that these are two different paragraphs.